Modules/Exercise05/Tests/Unit/Services/OrderServiceTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Modules\Exercise05\Tests\Unit\Services;
+
+use Modules\Exercise05\Services\OrderService;
+use Tests\TestCase;
+
+class OrderServiceTest extends TestCase
+{
+    /**
+     * @param $input
+     * @param $expected
+     * @dataProvider donotReceiveAtHomeAndHaveCoupon
+     * @dataProvider receiveAtHomeAndNoCoupon
+     * @dataProvider priceLessThanDiscountPotato
+     */
+    public function test_handle_discount_function($input, $expected)
```

Comment:
Trong provider này nên thêm có 1 case nữa check độ chính xác tới 2 chữ số thập phân vì cái round nó có thêm tham số thứ 2 là `2`
```
$infoBill['price'] = round(($detailOrder['price'] * 80) / 100, 2);
```
Cái này nice to have :smile: 

Nhưng mà nên có 1 case `price` bằng với `config('exercise05.price_has_discount_potato')` thì mình sẽ cover đưowcj case `>=`